File: MyTournamentAPI/OurTournamentAPI/Models/Goleadores.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace OurTournamentAPI.Models
{
    public class Goleadores
    {
        private int IDUsuario;
        private String NombreUsuario;
        private String NombreEquipo;
        private int Goles;

        public int IDUsuario1 { get => IDUsuario; set => IDUsuario = value; }
        public string NombreUsuario1 { get => NombreUsuario; set => NombreUsuario = value; }
        public string NombreEquipo1 { get => NombreEquipo; set => NombreEquipo = value; }
        public int Goles1 { get => Goles; set => Goles = value; }

        public Goleadores(int idusuario, String Nombreusuario, String Nombreequipo, int goles)
        {
            IDUsuario = idusuario;
            NombreUsuario = Nombreusuario;
            NombreEquipo = Nombreequipo;
            Goles = goles;
        }
        public Goleadores()
        {
        }
    }
}

File: MyTournamentApp/app/src/main/java/com/example/ourtournament/Objetos/Goleadores.java
package com.example.ourtournament.Objetos;

import android.widget.ImageView;

import java.sql.Date;

public class Goleadores {
    public int IDUsuario1;
    public String NombreUsuario1;
    public String NombreEquipo1;
    public int Goles1;

    public Goleadores(int IDUsuario, String NombreUsuario, String NombreEquipo, int goles)
    {
        IDUsuario1 = IDUsuario;
        NombreUsuario1 = NombreUsuario;
        NombreEquipo1 = NombreEquipo;
        Goles1 = goles;
    }
    public Goleadores()
    {
    }
}
